ui/src/views/compute/wizard/MultiNetworkSelection.vue

```diff
@@ -49,7 +49,8 @@
               v-for="network in validNetworks[record.id]"
               :key="network.id"
               :label="network.displaytext + (network.broadcasturi ? ' (' + network.broadcasturi + ')' : '')">
-              {{ network.displaytext + (network.broadcasturi ? ' (' + network.broadcasturi + ')' : '') }}
+              <div v-if="this.hypervisor === 'KVM'">{{ network.displaytext  + ' - ' + (network.id.slice(0,8)) }}</div>
```

Comment:
no idea about these constants. Would be better to make a new method with some doc

Reply:
Sure, I can create a separate method for generating this string?

Context: This is the label for network name in selection dropdown. Since displaytext can be non-unique, I added 8 characters of the UUID for differentiating in case of KVM. For VMWare we had broadcast URI from Vlan.